api/src/repositories/survey-repository.ts

```diff
@@ -1100,4 +1101,188 @@ export class SurveyRepository extends BaseRepository {
 
     return response.rowCount;
   }
+
+  /**
+   * @TODO jdsdoc
+   *
+   * @TODO should probably simplify this method to use knex? And accept [] as a valid response? ...
+   *
+   * @param {number} surveyId
+   * @return {*}  {Promise<GetPartnershipsData['indigenous_partnerships'][]>}
+   * @memberof SurveyRepository
+   */
+  async getIndigenousPartnershipsBySurveyId(
+    surveyId: number
+  ): Promise<GetPartnershipsData['indigenous_partnerships'][]> {
+    defaultLog.debug({ label: 'getIndigenousPartnershipsBySurveyId', surveyId });
+
+    const sqlStatement = SQL`
+      SELECT
+        fn.first_nations_id as id,
+        fn.name as first_nations_name
+      FROM
+        survey_first_nation_partnership sfnp
+      LEFT OUTER JOIN
+        first_nations fn
+      ON
+      sfnp.first_nations_id = fn.first_nations_id
+      WHERE
+      sfnp.survey_id = ${surveyId}
+      GROUP BY
+        fn.first_nations_id,
+        fn.name;
+    `;
+
+    const response = await this.connection.query(sqlStatement.text, sqlStatement.values);
```

Comment:
A couple of notes: Lets prefer the `.sql` query (its the same as `.query` but you can pass the whole `sqlStatement` to it rather than the text and values parts (trying to slowly deprecate `.query`).

We should pass a zod schema as the second parameter in all of these calls, even if its defined in-line. Still a good step in the right direction for response validation. The generic type can be removed then as well (in spots below that defined one), as the zod schema will ensure the response is typed.

Reply:
Yeah I caught the `.query` discrepancy just now. Will work to get some Zod schema incorporated